src/test/java/core/basesyntax/service/impl/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.CsvFileReader;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFileReaderImplTest {
+    private static final String INCORRECT_QUALITY_FRUIT_SHOP_CSV = "src/test/resources/"
+            + "incorrectQuality_fruit_shop.csv";
+    private static final String INCORRECT_OPERATION_FRUIT_SHOP_CSV = "src/test/resources/"
+            + "incorrectOperation_fruit_shop.csv";
+    private static final String CORRECT_FRUIT_SHOP_CSV = "src/test/resources/"
+            + "correct_fruit_shop.csv";
+    private static final CsvFileReader CSV_FILE_READER = new CsvFileReaderImpl();
```

Comment:
```suggestion
    private static final CsvFileReader csvFileReader = new CsvFileReaderImpl();
```
It's not a constant even though it's static final